lab3/src/task1/CandyBag.java

```diff
@@ -0,0 +1,69 @@
+package task1;
+
+import java.util.ArrayList;
+
+public class CandyBag {
+    private ArrayList<CandyBox> candies;
+
+    public CandyBag() {
+        this.candies = new ArrayList<>();
+        this.candies.add(
+                new Lindt(
```

Comment:
I kinda get what you tried to do, but because you don't have that many fields to initialize, you could have left them on a single line :)